src/plugins/home/public/application/components/guided_onboarding/getting_started.tsx

```diff
@@ -62,6 +63,8 @@ export const GettingStarted = () => {
 
   const onSkip = () => {
     trackUiMetric(METRIC_TYPE.CLICK, 'guided_onboarding__skipped');
+    // disable welcome screen on the home page
+    localStorage.setItem(KEY_ENABLE_WELCOME, JSON.stringify(false));
```

Comment:
nit: wouldn't `localStorage.setItem(KEY_ENABLE_WELCOME, 'false');` work as well?

Reply:
I actually find the boolean value more readable (also less error/typo prone)